.renovaterc.json

```diff
@@ -35,6 +35,7 @@
     {
       "groupName": "validator devDependencies",
       "matchPaths": ["validator/**"],
+      "excludePackageNames": ["amp-validator-webui"],
```

Comment:
This will have the undesired effect of ungrouping the updates but not disabling them. We'll likely need to add a new rule to disable updates altogether. I can send out a PR tomorrow.

Reply:
I just sent out #34202.